### DISCUSSION TITLE
libsql implementation compability

### DISCUSSION THREAD

**Discussion by c0ncentus** [tauri-apps/tauri#11689] [Ideas]
Hi,
[libSQL](https://www.youtube.com/watch?v=PGpL5hYpY1o) at https://github.com/tursodatabase/libsql
I watch this video it would be awesome to add libSQL because seems more tauri-spirit.

I dont want to implicate myslef so much in this suggestion.

**Comment by FabianLars** [comment]
Thanks for the suggestion, this is tracked here already https://github.com/tauri-apps/plugins-workspace/issues/1907 (no active plans right now)
